ntbs-service/Models/Notification.cs

```diff
@@ -1,14 +1,20 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
+using ntbs_service.Models.Enums;
 
 namespace ntbs_service.Models
 {
     public class Notification
     {
+        public Notification() {
+            NotificationStatus = Enums.NotificationStatus.Draft;
```

Comment:
This is a bit pedantic haha, but do you need to use the Enums namespace here considering you have the using statement above? Doesn't NotificationStatus.Draft just work?

Reply:
Visual Code wasn't happy with it yesterday :) now it works fine with just NotificationStatus.Draft